pygments/lexers/markup.py

```diff
@@ -555,7 +555,7 @@ def _handle_codeblock(self, match):
             (r'^(\s*)([0-9]+\.)( .+\n)',
             bygroups(Text, Keyword, using(this, state='inline'))),
             # quote
-            (r'^(\s*>\s)(.+\n)', bygroups(Keyword, Generic.Emph)),
+            (r'^(\s*[> ]+\s)(.+\n)', bygroups(Keyword, using(this, state='inline'))),
```

Comment:
Are you sure about his? `[> ]+` matches indentation, and I recall that in some Markdown dialects an indented block is a code block.